app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+
+    if user&.authenticate params[:session][:password]
```

Comment:
toán tử safe navigation `&.` có tác dụng gì?

Reply:
toán tử safe navigation có tác dụng tránh bị báo lỗi nếu đối tượng ta gọi tới để sử dụng các phương thức của nó bị nil